src/AddDiveWish.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Button from './Button.js';
+
+export default function AddListItem({ onAddDiveWish }) {
```

Comment:
I think I would name the component as you name the file.
```suggestion
export default function AddDiveWish({ onAddDiveWish }) {
```

Reply:
Ooops, I had to rename some components yesterday. Seems like I forgot that one